benchmarks/cpp/transformer.cpp

```diff
@@ -40,42 +39,43 @@ constexpr double kDropoutProb = 0.0, kSdpaProb = 0.0, kSdpaScale = 1e-3;
 // recommendations:
 // https://huggingface.co/docs/transformers/en/model_doc/gpt2#transformers.GPT2Config.initializer_range
 constexpr double kParamScale = 0.02;
-} // namespace
 
 // Return reduction tensor view and output of reduction
 void setupTransformerForward(Fusion* fusion, DataType dtype) {
-  Communicator* communicator_ = &Communicator::getInstance();
+  Communicator* communicator = &Communicator::getInstance();
 
-  const int64_t D = communicator_->size(); // number of devices
+  const int64_t D = communicator->size(); // number of devices
 
   auto model = std::make_unique<DistributedTransformer>(
       D, B, E, H, S, kDropoutProb, kSdpaProb);
 
   model->setupForward(fusion, dtype, /*sequence_parallel=*/false);
 }
 
-at::Tensor transformerShardTensor_Mesh(
-    at::Tensor tensor,
-    const int64_t axis,
-    const DeviceMesh& mesh,
-    Communicator* communicator_) {
-  const auto device_id = communicator_->deviceId();
+namespace {
+  at::Tensor shardTensor(
+      at::Tensor tensor,
+      const int64_t axis,
+      const DeviceMesh& mesh,
+      Communicator* communicator) {
+  const auto device_id = communicator->deviceId();
   return nvfuser::shardTensor(tensor, axis, mesh, device_id);
 }
+}
```

Comment:
```suggestion
} // namespace
```
When there are nested namespaces and when they span many lines of code, having a comment like this helps to figure out which namespace a particular line is in.